automation/project/Dependencies.scala

```diff
@@ -12,6 +12,9 @@ object Dependencies {
   val workbenchGoogleV = "0.10-d97f551"
   val workbenchGoogle: ModuleID = "org.broadinstitute.dsde.workbench" %% "workbench-google" % workbenchGoogleV excludeAll excludeWorkbenchModel
 
+  val workbenchServiceV = "0.0.1-alpha-b350837-SNAP"
```

Comment:
this is my snapshot. it only has Thurloe moved over

Reply:
Is this why "definitely don't merge"?